src/test/java/org/opentripplanner/middleware/ApiKeyManagementTest.java

```diff
@@ -63,8 +66,9 @@ public static void tearDown() {
         assumeTrue(isEndToEnd);
         // refresh API key(s)
         apiUser = Persistence.apiUsers.getById(apiUser.id);
-        apiUser.delete(false);
+        apiUser.delete();
         Persistence.adminUsers.removeById(adminUser.id);
+        setAuthDisabled(prevAuthState);
```

Comment:
+1

Reply:
Couldn't this just use `Auth0Connection#getDefaultAuthDisabled` instead of `prevAuthState`?